merge_test.go

```diff
@@ -1767,7 +1886,7 @@ func TestMergeArray(t *testing.T) {
 				So(err, ShouldBeNil)
 			})
 
-			Convey("throw an error when delete point cannot be found", func() {
+			Convey("don't throw an error when delete point cannot be found", func() {
```

Comment:
Yes, I have changed the behaviour to put it in sync with what `prune` does. If you don't find a key, you don't raise an exception. 

 I had a discussion this with Matthias. We should introduce like a global **strict mode** on Spruce, in which you will raise exceptions if something isn't found.